h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -58,6 +69,73 @@ public ComputationState(Job job, GLMParameters parms, DataInfo dinfo, BetaConstr
     _intercept = _parms._intercept;
     _nclasses = (parms._family == Family.multinomial||parms._family == Family.ordinal)?nclasses:1;
     _alpha = _parms._alpha[0];
+    if (_parms._HGLM) {
+      _sumEtaSquareConvergence = new double[2];
+      if (_parms._calc_like)
+        _likelihoodInfo = new double[4];
+    }
+  }
+  public void set_sumEtaSquareConvergence(double[] sumInfo) {
+    _sumEtaSquareConvergence = sumInfo;
+  }
+  
+  public void set_beta_HGLM(double[] beta, int startIdx, int len, boolean interceptFirst) {
```

Comment:
`len` is length of what ? By looking at this method and seeing `  _beta = new double[len];`, I'm super,confused. 

Please add JavaDoc here or use better naming. JavaDoc is fine.

Reply:
Added the description in JavaDoc.  Thanks.